packages/actor-query-parse-sparql/package.json

```diff
@@ -44,7 +44,11 @@
     "@comunica/bus-query-parse": "^4.0.2",
     "@comunica/context-entries": "^4.0.2",
     "@comunica/core": "^4.0.2",
-    "@traqula/engine-sparql-1-2": "^0.0.1-alpha.148",
-    "sparqlalgebrajs": "^5.0.1"
+    "@traqula/algebra-sparql-1-2": "^0.0.15",
+    "@traqula/parser-sparql-1-2": "^0.0.15",
+    "@traqula/rules-sparql-1-2": "^0.0.15"
+  },
+  "devDependencies": {
```

Comment:
This is incorrect. Type packages must always be actual deps.

Reply:
Oh, why?